homeassistant/components/wyoming/__init__.py

```diff
@@ -47,51 +47,29 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     entry.async_on_unload(entry.add_update_listener(update_listener))
 
     if (satellite_info := service.info.satellite) is not None:
-        # Create satellite device, etc.
-        item.satellite = _make_satellite(hass, entry, service)
-
-        # Set up satellite sensors, switches, etc.
-        await hass.config_entries.async_forward_entry_setups(entry, SATELLITE_PLATFORMS)
+        # Create satellite device
+        dev_reg = dr.async_get(hass)
+
+        # Use config entry id since only one satellite per entry is supported
+        satellite_id = entry.entry_id
+        device = dev_reg.async_get_or_create(
```

Comment:
Shouldn't we always create a device (with device_type="service" btw), and put all service entities linked to it.

Reply:
(for a future PR)

Reply:
"Always" as in even if there isn't a satellite present (i.e., for a TTS service)?